.buildkite/build_pipeline.yml

```diff
@@ -25,6 +25,9 @@ steps:
             value: ""
         hint: "Should we ignore checking broken links? Should we allow to run the build without failing if there's a broken link? Ignoring broken links is dangerous not just because bad links will leak into the public site but because subsequent builds and pull requests that do not fix the links fail."
   - wait
+  - label: "Full rebuild or incremental build?"
+    if: build.source == "schedule"
+    command: ".buildkite/scripts/compare_commits.sh"
```

Comment:
@nkammah why does L30 not beet `bash` in the command but L35 does?

Reply:
We specify the script interpreter in the first line of the script [here ](https://github.com/elastic/docs/pull/2931/files#diff-e1a7c7e62391e5c40e7fa4b5cb419d9d1915d056ea8f30192cf6515cb8ff43d9R1) - that's what we call the "shebang". If the script is executable (ex. running "chmod a+x" on it), the pipeline will know how to call it.

As for the discrepancy on L35, it could be legacy, or that we explicitly specified it at the time. Both approaches will work.